lib/nexthop.h

```diff
@@ -77,7 +77,6 @@ struct nexthop {
 
 	uint8_t flags;
 #define NEXTHOP_FLAG_ACTIVE     (1 << 0) /* This nexthop is alive. */
```

Comment:
Hi,
struct route_entry {
	/* Nexthop structure (from RIB) */
	struct nexthop_group ng;
	/* Nexthop group from FIB (optional) */
	struct nexthop_group fib_ng;
}
Query: If we have the above 2 groups in route_entry, is there any need for NEXTHOP_FLAG_FIB. I assume that fib_ng will have all the nexthops installed in hardware. Is it true ?

Reply:
I mean that nexthops from 2nd list (struct nexthop_group fib_ng) would have been marked with NEXTHOP_FLAG_FIB in 1st list.(struct nexthop_group ng). So, for our display commands etc, we can use 1st and 2nd list effectively so that flag NEXTHOP_FLAG_FIB is not needed.

Reply:
It's not super-useful to engage with a closed PR: this isn't going to be worked on in the future. That said, there are some cases where a routing protocol may include a set of nexthops with a route, but only a subset is actually installed and active: a failed link, or a nexthop destination that becomes unreachable, for example.